src/components/MovieInfo/MovieInfo.js

```diff
@@ -1,28 +1,39 @@
 
 import React, { Component } from 'react';
-import testDetails from '../MovieInfo/mockMovieView';
+import { fetchSingleMovieDetails } from '../../apiCalls';
 import './MovieInfo.css';
 
 class MovieInfo extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movie: testDetails.movie
-    };
+  constructor() {
+    super();
+    this.state = {};
   }
 
-  render() {
-    const { movie } = this.state;
+  componentDidMount() {
+    fetchSingleMovieDetails(this.props.selectedMovieId)
+        .then(data => this.setState(data.movie))
+        .catch()
+  }
 
+  render() {
+    let dollarConversion = new Intl.NumberFormat('en-US', {
+      style: 'currency',
+      currency: 'USD'
+    });
     return (
       <section>
-        <img src={movie.poster_path} alt="Movie Poster" />
+        <img src={this.state.poster_path} alt="Movie Poster" />
         <div className="moreInfo">
-          <p>{movie.overview}</p>
-          <p>{movie.average_rating}</p>
-          <p>{movie.genres[0]}</p>
-          <p>{movie.tagline}</p>
-          <p>{movie.overview}</p>
+        <p>Title: {this.state.title}</p>
+        {/* <p>{this.state.backdrop_path}</p> */}
+        <p>Release Date: {this.state.release_date}</p>
+        <p>Overview: {this.state.overview}</p>
+        <p>Genre: {this.state.genres}</p>
+        <p>Budget: {dollarConversion.format(this.state.budget)}</p>
+        <p>Revenue: {dollarConversion.format(this.state.revenue)}</p>
+        <p>Duration: {this.state.runtime}</p>
+        <p>Tagline: {this.state.tagline}</p>
+        <p>Rating: {this.state.average_rating}</p>
```

Comment:
Anytime I see a lot of similar dot notation, it indicates an opportunity to do some destructuring so you can more simply and cleanly access each piece of data.  Not required, but something you might want to try out here.